test/src/test/java/org/corfudb/integration/CorfuReplicationUpgradeIT.java

```diff
@@ -937,4 +939,46 @@ private void verifyVersion(CorfuStore corfuStore, String expectedVersion, boolea
         Assert.assertEquals(isUpgraded, actualUpgradedFlag);
         log.info("Verified version");
     }
+
+    /**
+     * Code coverage test for the simple LRRollingUpgradeHandler to test if we are able to successfully
+     * 1. Simulate startRollingUpgrade()
+     * 2. verify that rolling upgrade is detected
+     * 3. simulate endRollingUpgrade()
+     * 4. verify that data migration and rolling upgrade end is detected
+     *
+     * @throws Exception
+     */
+    @Test
+    public void testLocalClusterRollingUpgrade() throws Exception {
+        log.info(">> Setup replication for testing during rolling upgrade of active cluster");
```

Comment:
Shama's source/sink change is in the `log_replication_v2` branch once it merges to master we would fix up all the usages including this one.

Reply:
The rolling upgrade handling is independent of source or sink so I only added one of the two and since they extend the base case which has the implementation, it still gives code coverage.